### DISCUSSION TITLE
What does the “A>” sign in the gutter indicate?

### DISCUSSION THREAD

**Discussion by sideshowbarker** [prabirshrestha/vim-lsp#1546] [Q&A] [answered]
I see an `A>` sign in the gutter next to a particular line, and it mostly seems to remain at the line for some time. And then when I make an edit to a later line, that `A>` sign generally moves to the line where I made my edit. But sometimes it doesn’t. Sometimes it shows up again on an even-later line. And sometimes it disappears entirely and then eventually reappears at some other line.

And vim never shows and diagnostic message associated with that sign. And calling `:LspDocumentDiagnostics` doesn’t list any message for it.

So I remain a bit baffled about what that `A>` sign is indicating.

**Comment by sideshowbarker** [comment, marked as answer]
Found the answer in [`:help lsp_document_code_action_signs_enabled`](https://github.com/prabirshrestha/vim-lsp/blob/f7ccf006df1aefd327c0e2c55cc8632a2db577c1/doc/vim-lsp.txt#L788):

> Enables signs for code actions.
>
> `LspCodeActionText` defaults to `A>`.

So, the `A>` sign marks a line with a suggested code action, and `let g:lsp_document_code_action_signs_enabled = 0` can be used to disable the display of that sign.

**Reply by Starou** [reply]
Thanks, but how do you display the suggested action please?
